test/dumpfile.sql

```diff
@@ -9,7 +9,7 @@ CREATE TABLE `articles` (
   `extra` text,
   `thumb` varchar(1000) DEFAULT NULL,
   `author_id` bigint(20) DEFAULT NULL,
-  `is_private` tinyint(1) DEFAULT 0,
+  `is_private` tinyint(1) DEFAULT 0 NOT NULL,
```

Comment:
v2.0.2 and v2.0.3 will throw LeoricValidationError in such column settings, because the attribute.defaultValue is not present, but the values were validated before create.